src/misc/setup.cpp

```diff
@@ -1234,7 +1234,33 @@ bool CommandLine::FindExist(const char *name, bool remove)
 	return true;
 }
 
-bool CommandLine::FindInt(const char *name, int &value, bool remove)
+// Checks if any of the command line arguments are found in the anterior
+// arguments and come before any of the posterior arguments. If none of the
+// command line arguments are found in the anterior then false is returned.
+bool CommandLine::ExistsPriorTo(const std::list<std::string>& anterior,
+                                const std::list<std::string>& posterior) const
```

Comment:
minor (and only a preference 😄): maybe call these `pre_args` and `post_args`?

Reply:
Yes; much better 👍!